python/ray/tests/test_gpu_objects_gloo.py

```diff
@@ -422,5 +426,99 @@ def test_tensor_extracted_from_tensordict_in_gpu_object_store(ray_start_regular)
     assert torch.equal(ret_val_src[1], td["reward"])
 
 
+def test_duplicate_objectref_transfer(ray_start_regular):
+    """
+    This test checks for a race condition when the same GPU object ref 
+    is passed to the same actor multiple times.
```

Comment:
I think it's fine not to mention the potential bug ("race condition") because probably others will not have context for it anyway.